api/core/models.py

```diff
@@ -136,12 +136,12 @@ def _get_project(self) -> typing.Optional["Project"]:
 def get_history_user(
     instance: typing.Any, request: HttpRequest
 ) -> typing.Optional["FFAdminUser"]:
-    from api_keys.user import APIKeyUser
-
-    try:
-        return None if isinstance(request.user, APIKeyUser) else request.user
-    except AttributeError:
-        return None
+    return (
+        request.user
+        if hasattr(request, "user")
+        and not hasattr(request.user, "is_master_api_key_user")
```

Comment:
I think I'd rather see these as `getattr` instead of `hasattr`?

Reply:
Do you mean something like the current version?

Reply:
Yep, looks good.